### ISSUE TITLE
console: deleted workspace still shows up on workspace list

### ISSUE THREAD

**Issue by nguyendon** [anomalyco/sst#5114]
The offending workspace is named `ymcu`.

`https://console.sst.dev/ymcu`

**Comment by nguyendon** [comment]
It looks like I'm still getting notifications from this workspace as well.

**Comment by jayair** [comment]
@nguyendon I think we fixed the notifications issue. Are you still getting them?

**Comment by nguyendon** [comment]
@jayair i was still getting them but I just went into the app and deleted the autodeploy / notifications and that got rid of them. They weren't destroyed as part of the delete workspace workflow though.

**Comment by jayair** [comment]
Wait you deleted the workspace but you were still able to go back into the app?

**Comment by nguyendon** [comment]
@jayair yup I am, there isn't a link to it in console but I can visit the link directly at `https://console.sst.dev/<org>/<project/autodeploy`

**Comment by Drakeoon** [comment]
I have the same issue on my end. I can add that I see the deleted workspace in the "Search" popup in the console. Can also access it through URL as @nguyendon mentioned (it displays the "Connect an AWS account" screen)
